e2e/paths/loginAsHmctsAdmin_test.js

```diff
@@ -0,0 +1,19 @@
+const config = require('../config.js');
+let caseId;
+
+Feature('Login as hmcts admin');
+
+Before(async (I, caseViewPage) => {
+  I.logInAndCreateCase(config.localAuthorityEmail, config.localAuthorityPassword, config.eventSummary, config.eventDescription);
+  caseId = await I.grabTextFrom('h2');
+  caseViewPage.goToNewActions(config.applicationActions.submitCase);
+  I.click('.button[type=submit]');
```

Comment:
We should use `/e2e/pages/createCase/addEventSummary.js#submitCase` here with `summary` and `description` set to default `''`

Reply:
I don't think I can use this. The test will fail as there is no event description or event summary text fields on the page with the submit button